tools/cldr-code/src/main/java/org/unicode/cldr/util/ExtraPaths.java

```diff
@@ -101,14 +101,19 @@ private void addPaths(NameType nameType) {
         }
 
         private void adjustCodeSet(Set<String> codes, NameType nameType) {
-            if (nameType == NameType.LANGUAGE) {
-                codes.remove(LocaleNames.ROOT);
-                codes.addAll(
-                        List.of(
-                                "ar_001", "de_AT", "de_CH", "en_AU", "en_CA", "en_GB", "en_US",
-                                "es_419", "es_ES", "es_MX", "fa_AF", "fr_CA", "fr_CH", "frc",
-                                "hi_Latn", "lou", "nds_NL", "nl_BE", "pt_BR", "pt_PT", "ro_MD",
-                                "sw_CD", "zh_Hans", "zh_Hant"));
+            switch (nameType) {
+                case LANGUAGE:
+                    codes.remove(LocaleNames.ROOT);
+                    codes.addAll(
+                            List.of(
+                                    "ar_001", "de_AT", "de_CH", "en_AU", "en_CA", "en_GB", "en_US",
+                                    "es_419", "es_ES", "es_MX", "fa_AF", "fr_CA", "fr_CH", "frc",
+                                    "hi_Latn", "lou", "nds_NL", "nl_BE", "pt_BR", "pt_PT", "ro_MD",
+                                    "sw_CD", "zh_Hans", "zh_Hant"));
+                    break;
+                case TERRITORY:
+                    codes.addAll(List.of("XA", "XB"));
```

Comment:
This is not adding an alt="variant" version of the paths.

Reply:
Right. This corresponds to these two lines removed from XMLSource:
```
            addFallbackCode(NameType.TERRITORY, "XA", "XA");
            addFallbackCode(NameType.TERRITORY, "XB", "XB");`
```
Neither the old nor the new code involves alt. What change are you requesting?

Reply:
Note that the old code is calling this method:

```
        private static void addFallbackCode(NameType nameType, String code, String value) {
            addFallbackCode(nameType, code, value, null);
        }
```

Here `null` corresponds to `alt` in this method:

```
        private static void addFallbackCode(
                NameType nameType, String code, String value, String alt) {
            String fullpath = nameType.getKeyPath(code);
            String distinguishingPath = addFallbackCodeToConstructedItems(fullpath, value, alt);
            if (nameType == NameType.SCRIPT || nameType == NameType.TERRITORY) {
                allowDuplicates.put(distinguishingPath, code);
            }
        }
```

Reply:
Also note that this PR has 2 commits, and the first had a mistake:
```
                    addAltPath("XA", "variant", nameType);
                    addAltPath("XB", "variant", nameType);
```
That was corrected in the 2nd commit.

Reply:
Ah, right. We need both the main path and the alternate.